src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.PackageDependencyResolution.targets

```diff
@@ -170,6 +170,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ResolvePackageDependencies
       ProjectPath="$(MSBuildProjectFullPath)"
       ProjectAssetsFile="$(ProjectAssetsFile)"
+      References="@(Reference)"
```

Comment:
@natidea We need the metadata from the Reference (that is, whether or not it is a façade, and the NuGetPackageId and NuGetPackageVersion).

I'll take a look at the ordering.